test/e2e/kubelet_deletes.go

```diff
@@ -0,0 +1,146 @@
+/*
```

Comment:
It depends. There are also "service.go" and "service_latency.go". 

I think kubelet is a big enough component that grouping all the tests in one file may be hard to read in the future. Given that this is easy to change and we don't have other tests yet, I think it might be okay to keep the filename. I can change if you insist.